npm/private/npm_package_store.bzl

```diff
@@ -189,14 +188,16 @@ def _npm_package_store_impl(ctx):
 
     # the path to the package store location for this package
     # "node_modules/{package_store_root}/{package_store_name}/node_modules/{package}"
-    package_store_directory_path = paths.join("node_modules", utils.package_store_root, package_store_name, "node_modules", package)
+    package_store_directory_path = "node_modules/{}/{}/node_modules/{}".format(utils.package_store_root, package_store_name, package)
 
     if ctx.attr.src and NpmPackageInfo in ctx.attr.src:
         # output the package as a TreeArtifact to its package store location
         if ctx.label.workspace_name:
-            expected_short_path = paths.join("..", ctx.label.workspace_name, ctx.label.package, package_store_directory_path)
+            expected_short_path = "../{}/{}/{}".format(ctx.label.workspace_name, ctx.label.package, package_store_directory_path)
```

Comment:
Is it possible `ctx.label.package` is not set, and `paths.join` was handling that before?